app/src/main/java/org/simlar/service/liblinphone/LinphoneManager.java

```diff
@@ -611,6 +605,11 @@ public void onDtmfReceived(@NonNull final Core lc, @NonNull final Call call, fin
 	{
 		Lg.w("onDtmfReceived number=", new CallLogger(call), " dtmf=", dtmf);
 	}
+	@Override
```

Comment:
There is a new line missing.

Reply:
Thanks again @jritzerfeld!
I have pushed a new commit fixing this.